diff --git a/src/Controller/EmployesController.php b/src/Controller/EmployesController.php
index 3bd516b..0eb7156 100644
--- a/src/Controller/EmployesController.php
+++ b/src/Controller/EmployesController.php
@@ -8,6 +8,7 @@ use App\Form\EmployesType;
 use App\Repository\EmployesRepository;
 use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\File\Exception\FileException;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
diff --git a/public/uploads/employes/.gitkeep b/public/uploads/employes/.gitkeep
new file mode 100644
index 0000000..e69de29
